app/src/main/java/com/ucmobiledevelopment/freeways/dto/IncidentDTO.kt

```diff
@@ -15,5 +15,5 @@ data class IncidentDTO(
     val TOTALVEHICLES: String? = "",
     val TWAY_ID: String? = "",
     val TWAY_ID2: String? = "",
-    val VE_FORMS: String? = ""
+    val VE_FORMS: String? = "",
```

Comment:
Trailing commas, while not required, are an easy way to reduce future technical debt: https://kotlinlang.org/docs/coding-conventions.html#trailing-commas